Dangerfile

```diff
@@ -26,3 +26,19 @@ if github.pr_body.include?("* [ ]") && !github.pr_title.include?("[WIP]")
 end
 
 message "Pull Request is marked as Work in Progress" if github.pr_title.include? "[WIP]"
+
+junit.parse "output.xml"
+junit.report
```

Comment:
Also we no longer need this line to show the failed test in a separate table as we are displaying it manually.